examples/dihedral/dihedral.yml

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env yason.py
+#
+# Combined MC and MD moves on a four-bead molecule
+# with 3 harmonic bonds, 2 harmonic bond angle, and 1 dihedral angle.
+#
+# You may disable either MD or MC to verify that they
+# individually give the same bead<->bead distribution
+#
+energy:
+    - bonded: {}
+
+atomlist:
+    - a: {mw: 1, dp: 5}
+    - b: {mw: 1, dp: 5}
+    - c: {mw: 1, dp: 5}
+    - d: {mw: 1, dp: 5}
+
+moleculelist:
+    - fourbody:
+        atomic: false
+        atoms: [a, b, c, d]
+        structure:
+          - a: [-9.80, -3.11, 7.57]
+          - b: [-9.56, -3.85, 8.26]
+          - c: [-8.10, -4.43, 7.78]
+          - d: [-8.38, -3.05, 7.16]
+        bondlist:
+          - harmonic: {k: 10, req: 1, index: [0,1]}
+          - harmonic: {k: 10, req: 1, index: [1,2]}
+          - harmonic: {k: 10, req: 1, index: [2,3]}
+          - harmonic_torsion: {k: 10, aeq: 90, index: [0,1,2]}
+          - harmonic_torsion: {k: 10, aeq: 45, index: [1,2,3]}
+          - periodic_dihedral: {k: 5, n: 2, phi: 180, index: [0,1,2,3]}
+
+insertmolecules:
+    - fourbody: {N: 1}
+
+moves:
+    - langevin_dynamics: # Langevin dynamics
+        nsteps: 10
+        integrator: {time_step: 0.002, friction: 5}
+#    - transrot: # Monte Carlo
+#        molecule: fourbody
+
+analysis:
+    - systemenergy: {file: energy.dat, nstep: 100, nskip: 1}
+#    - reactioncoordinate:
+#          file: distance.dat.gz
+#          nstep: 1
+#          nskip: 10
+#          type: molecule
+#          index: 0
+#          property: atomatom
+#          indexes: [0,2]
+#          dir: [1,1,1]
+    - atomrdf: {file: rdf.dat, nstep: 1, dr: 0.1, name1: a, name2: d, nskip: 1, dim: 1}
+    - savestate: {file: state.json}
+
+geometry: {type: cuboid, length: 20}
+mcloop: {macro: 1, micro: 5000000}
```

Comment:
this might be a bit long for a test; aim seconds in terms of run time

Reply:
Fixed with 65a4c62.